examples/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "examples"
+version = "0.0.0"
+publish = false
+edition = "2018"
+
+[dev-dependencies]
+pyo3 = { version = "0.15.1", path = "..", features = ["auto-initialize", "extension-module"] }
+
+[[example]]
+name = "decorator"
+path = "decorator/src/lib.rs"
```

Comment:
That depends on what you mean with "use in CI". If you run `cargo fmt --all -- --check` for example, it will check the contents of `decorator/src/lib.rs` as well.

Reply:
You're right; however the `pytests/` content was not getting checked. I've tweaked the makefile, which should cover it.